core/claimmanager.go

```diff
@@ -340,7 +337,7 @@ func shouldVerifySegment(seqNum int64, start int64, end int64, blkNum int64, blk
 	seqNumTmp := make([]byte, 8)
 	binary.PutVarint(seqNumTmp, seqNum)
 	seqNumB := make([]byte, 32)
-	copy(seqNumB[24:], blkNumTmp)
+	copy(seqNumB[24:], seqNumTmp)
```

Comment:
Ah...so this function could've been returning a lot of false positives for segments to be verified